docker/Dockerfile.demo_vitis_ai

```diff
@@ -18,7 +18,7 @@
 # CI docker VAI env
 FROM xilinx/vitis-ai:latest
 
-RUN apt-get update --fix-missing
+RUN apt-get update --fix-missing && apt-get install -y gpg-agent
```

Comment:
To save same space on your image, Docker recommends a little boilerplate when using `apt-get`. I suggest adding that recommended `&& rm -rf /var/lib/apt/lists/*` in the end here.

You can read about it here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Thanks for suggestion @leandron . I moved the gpg-agent package installation to the  install/ubuntu_install_vitis_ai_core.sh script instead to clear up the dockerfile.

Reply:
Ack, see comment below.